modules/virt-configuring-cluster-dpdk.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-cluster-dpdk_{context}"]
+= Configuring a cluster for DPDK workloads
+
+Use this procedure to configure an {product-title} cluster to run Data Plane Development Kit (DPDK) workloads.
+
+:FeatureName: Configuring a cluster for DPDK workloads
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the SR-IOV Network Operator.
+* You have installed the Node Tuning Operator.
+
+.Procedure 
+. Map your compute nodes topology to determine which Non-Uniform Memory Access (NUMA) CPUs will be isolated for DPDK applications and which ones will be reserved for the OS.
```

Comment:
I know in IBM style we try to avoid the future tense and passive voice as much as we can, which this sentence has a bit of. It still reads fine to me, and I can't think of a suggestion, but I just wanted to make you aware 😄.

Reply:
Reworded to use present tense